Dockerfile

```diff
@@ -6,8 +6,18 @@ LABEL authors="Phil Ewels, Rickard Hammarén" \
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 
+
+RUN mkdir /opt/scripts
+
+COPY ./bin/* /opt/scripts/
+
+RUN find /opt/scripts/ -type f -iname "*.py" -exec chmod +x {} \; && \
+    find /opt/scripts/ -type f -iname "*.r"   -exec chmod +x {} \; && \
+    find /opt/scripts/ -type f -iname "*2bed" -exec chmod +x {} \;
+
 # Add conda installation dir to PATH (instead of doing 'conda activate')
 ENV PATH /opt/conda/envs/nf-core-rnaseq-1.4.3dev/bin:$PATH
+ENV PATH /opt/scripts:$PATH
 
 # Dump the details of the installed packages to a file for posterity
 RUN conda env export --name nf-core-rnaseq-1.4.3dev > nf-core-rnaseq-1.4.3dev.yml
```

Comment:
Modification to address the same issue as described [here](https://github.com/TheJacksonLaboratory/atacseq/pull/3/files#r419120532)